src/utils/getFilterTodos.ts

```diff
@@ -0,0 +1,16 @@
+import { FILTERS } from '../types/Filters';
+import { Todo } from '../types/Todo';
+
+export function getfilteredTodos(todos: Todo[], filter: string) {
+  let preperedTodos = [...todos];
+
+  switch (filter) {
+    case FILTERS.active:
+      preperedTodos = preperedTodos.filter(todo => !todo.completed);
+      break;
+    case FILTERS.completed:
+      preperedTodos = preperedTodos.filter(todo => todo.completed);
+  }
+
+  return preperedTodos;
+}
```

Comment:
```suggestion
export function getfilteredTodos(todos: Todo[], filter: string) {
  switch (filter) {
    case FILTERS.active:
      return todos.filter(todo => !todo.completed);
    case FILTERS.completed:
      return todos.filter(todo => todo.completed);
    default:
      return todos;
  }
}
```
- You don't need to create a copy as `filter` doesn't mutate an array.
- You should always have the default case in the switch